packages/ui/src/primitives/tailwind/Button/index.tsx

```diff
@@ -32,7 +32,16 @@ export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   endIcon?: ReactNode
   children?: ReactNode
   size?: 'small' | 'medium' | 'large'
-  variant?: 'primary' | 'outline' | 'danger' | 'success' | 'pink' | 'transparent' | 'onboarding' | 'sidebar'
+  variant?:
```

Comment:
This is getting to a lot of variants, are they all actually used?

Reply:
Checking all variants are being used except pink, so I just removed it